pkgs/os-specific/linux/bionic-prebuilt/default.nix

```diff
@@ -0,0 +1,113 @@
+{ stdenvNoCC, lib, fetchurl, fetchzip, pkgs
+}:
+let
+
+  prebuilt_crt = fetchzip {
+    url =  "https://android.googlesource.com/platform/prebuilts/gcc/linux-x86/aarch64/aarch64-linux-android-4.9/+archive/refs/heads/ndk-release-r23/lib/gcc/aarch64-linux-android/4.9.x.tar.gz";
+    sha256 = "1dshjv1557kf30nq4la4bph25ccpb713m9fvwi6w7wmvk0wgdc1c";
+    stripRoot = false;
+  };
+
+  prebuilt_libs = fetchzip {
+    url = "https://android.googlesource.com/platform/prebuilts/ndk/+archive/refs/heads/ndk-release-r23/platform/platforms/android-30/arch-arm64/usr/lib.tar.gz";
+    sha256 = "1k6dc4rszw4qbzbnmfavqhbvmz06y5p5fwc6zjgwhdzxba14baqd";
+    stripRoot = false;
+  };
+
+  prebuilt_ndk_crt = fetchzip {
+    url = "https://android.googlesource.com/toolchain/prebuilts/ndk/r23/+archive/refs/heads/master/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/lib/aarch64-linux-android/30.tar.gz";
+    sha256 = "sha256-KHw+cCwAwlm+5Nwp1o8WONqdi4BBDhFaVVr+7GxQ5uE=";
+    stripRoot = false;
+  };
+
+  ndk_support_headers = fetchzip {
+    url ="https://android.googlesource.com/platform/prebuilts/clang/host/linux-x86/+archive/refs/heads/ndk-release-r23/clang-r383902b/include/c++/v1/support.tar.gz";
+    sha256 = "0314caslxh6ngx4mf28f5vcfk332z7ds2szx6w4wr7p3s3f4xlv4";
+    stripRoot = false;
+  };
+
+  kernelHeaders = pkgs.makeLinuxHeaders {
+    arch = "arm64";
```

Comment:
`linuxArch` would be `arm64` anyways, so why do you need this to be a parameter?